pkgs/top-level/all-packages.nix

```diff
@@ -26782,6 +26782,11 @@ with pkgs;
 
   gosec = callPackage ../development/tools/gosec { };
 
+  gotraceui = callPackage ../development/tools/gotraceui {
+    inherit (darwin.apple_sdk) MacOSX-SDK;
```

Comment:
Why do we need `MacOSX-SDK`?

Reply:
Like I said, I copied the Darwin dependencies from Gio's Nix flake, the GUI framework I use. I know nothing about macOS development myself. It's possible that the requirement is superfluous, but I have no way of testing that.

Reply:
I think it would then be better to remove the part about Darwin support in this PR, and try to add it later. At least the PR is not mergeable with a red X in ofborg checks.

Reply:
That's fair, I've removed the Darwin bits.